surveyor/surveyor_test.go

```diff
@@ -214,31 +212,6 @@ func TestSurveyor_Reconnect(t *testing.T) {
 	}
 }
 
-func TestSurveyor_NoSystemAccount(t *testing.T) {
```

Comment:
the changes in nats to always have a system account seem to have broken this test in weird ways so I just killed it for now

Reply:
OK.